processor/src/main/java/org/mapstruct/ap/internal/model/common/Assignment.java

```diff
@@ -16,13 +16,11 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.mapstruct.ap.internal.model.assignment;
+package org.mapstruct.ap.internal.model.common;
```

Comment:
Maybe instead of using `Assignment` in the `common` package we could use the `SourceRHS`. I think that in that case no change in a package will be needed. What do you think?